corehq/apps/enterprise/static/enterprise/js/enterprise_dashboard.js

```diff
@@ -191,30 +191,35 @@ hqDefine("enterprise/js/enterprise_dashboard", [
     }
 
     $(function () {
```

Comment:
would be nice to make this an es6 / esm module and use `js_entry`

Reply:
Also, since this entry point will be used for both `enterprise_dashboard` and `security_watchtower`, would be good to rename the file so that it's clear it's not specific to either. e.g. `enterprise_dashboard` => `enterprise_report_tiles`?

Alternative...rename enterprise dashboard page to project health metrics?

Reply:
Or "Project Overview"?

Reply:
Addressed in f35b96fd89e34586e9b3d87af73cf94ce3eeeb34